src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -461,7 +461,7 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
       .then(updatedRequest => {
         if (this._config.useSourceTag) {
           const source = document.createElement('source');
-          const mimetype = this._sourceObj.mimetype.toLowerCase();
+          const mimetype = this._sourceObj ? this._sourceObj.mimetype.toLowerCase() : 'application/dash+xml';
```

Comment:
we can't put arbitrary value here, it must be coming from the source, if it is HLS but no mimetype is provided in config it might cause an error if incorrect one is provided

Reply:
since there is an option that sourceobj is empty what do you suggest to put? because the original change cause errors in the test because of it.

Reply:
U need to test it, I guess if there is no mimetype you should avoid setting the type all together
what are the errors if you set null? and are they in webos or also on web?